docs_src/getting-started/random-generator-devices.yaml

```diff
@@ -0,0 +1,15 @@
+deviceList:
+  - name: RandNum-Device01
+    profileName: RandNum-Device
+    description: "Random Number Generator Device"
+    labels:
+      - test
+      - random
+    protocols:
+      other:
```

Comment:
Will the protocol name being in lowercase cause any issues?

Reply:
Nope, because it is lower case here in the YAML attribute
https://github.com/edgexfoundry/go-mod-core-contracts/blob/main/dtos/device.go#L24

Reply:
Sorry, I am referring to `other`. Since it it just a map of strings, the key case is important. So it matters if anything uses that data or not.

Reply:
It failed with `Other` so I switched it to `other`. 
Consistent with existing device definition file here:
https://github.com/edgexfoundry/device-sdk-go/blob/main/example/cmd/device-simple/res/devices/simple-device.yml#L8

BTW, the TOML marshaling wasn't case sensitive.